Client/mods/deathmatch/logic/CPacketHandler.cpp

```diff
@@ -3902,14 +3902,19 @@ void CPacketHandler::Packet_EntityAdd(NetBitStreamInterface& bitStream)
                         vecVertices[i].fX = sX;
                         vecVertices[i].fY = sY;
                     }
+
+                    bool bShallow;
```

Comment:
bShallow should be initialized to a default value

Reply:
Thanks. This has now been changed.